superset-frontend/src/components/TimezoneSelector/TimezoneSelector.test.tsx

```diff
@@ -55,49 +77,65 @@ it('use the default timezone when an invalid timezone is provided', async () =>
   expect(onTimezoneChange).toHaveBeenLastCalledWith('Africa/Abidjan');
 });
 
-it.skip('can select a timezone values and returns canonical value', async () => {
+test('render timezones in correct oder for standard time', async () => {
```

Comment:
nit (if you happen to need to push more changes):
```suggestion
test('render timezones in correct order for standard time', async () => {
```

Reply:
Looks like I don't... Will fix it in some other PR.